### PR TITLE
Honor envs dirs

### PR DESCRIPTION
**Pull request by edisongustavo**

### CODE REVIEW

File: tests/test_environment_scripts.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def test_render_activate_and_deactivate_scripts_cmd(single_values, multiple_valu
     assert render_activate_script(single_values, "cmd") == textwrap.dedent("""\
         @echo off
         set CONDA_DEVENV_BKP_VALUE=%VALUE%
-        set VALUE="value"
+        set VALUE=value
```
Comment (nicoddemus) [inline tests/test_environment_scripts.py]: Is this intentional?

Comment (edisongustavo) [inline tests/test_environment_scripts.py]: Yes, I broke the build at 0.2 :)

Comment (nicoddemus) [inline tests/test_environment_scripts.py]: I thought that was already on `master`...

Comment (edisongustavo) [inline tests/test_environment_scripts.py]: Yeah, shame on me :(

Comment (gqmelo) [inline tests/test_environment_scripts.py]: I had commented as a review, but did not publish it.
On Windows this should actually be ```set "VALUE=value"```

File: conda_devenv/devenv.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ def render_activate_script(environment, shell):
                 value = pathsep.join(value) + pathsep + "%{variable}%".format(variable=variable)
 
             script.append("set CONDA_DEVENV_BKP_{variable}=%{variable}%".format(variable=variable))
-            script.append("set \"{variable}={value}\"".format(variable=variable, value=value))
+            script.append("set {variable}={value}".format(variable=variable, value=value))
```
Comment (gqmelo) [inline conda_devenv/devenv.py]: We should not remove the quotes. And actually we need to add quotes for all variables in all shells.
Maybe we create an issue to fix that?

Comment (nicoddemus) [inline conda_devenv/devenv.py]: I think it is simple enough and we should create a PR right away. :)

Comment (edisongustavo) [inline conda_devenv/devenv.py]: Yeah

img

**Pull request merged**